root/wrapper.html

```diff
@@ -79,7 +79,9 @@
         <%- END %>
         <%- IF need_less %>
         <script src="/static/js/less.min.js" type="text/javascript"></script>
+        <link rel="preload" href="/static/fonts/fa-regular-400.woff2" as="font" />
         <%- END %>
+        <link rel="preconnect" href="//www.google-analytics.com" />
```

Comment:
I don't see any value in adding this. The google analytics script is set up immediately after this, and it's async so it doesn't block any other part of the page.

Reply:
Agreed.